### ISSUE TITLE
Adding new provider documentation for OpenVZ

### ISSUE THREAD

**Issue by mrjcleaver** [fog/fog.github.com#32]
Re: https://github.com/fog/fog/issues/3405 - I noticed the OpenVZ is missing at http://fog.io/about/provider_documentation.html.  

What are your requirements for documentation? I didn't build the driver - @jedi4ever did - but I can quite believe Patrick is extremely busy and that this may not be a priority for him.

I don't have any experience with fog.io, and not much with OpenVZ. What's the minimum I need to do to get the documentation to an acceptable point as a PR?

I suppose just adding OpenVZ to that matrix is a start, but would that suffice?

Thanks kindly,
  Martin.

**Comment by geemus** [comment]
Yeah, adding to the matrix sounds like a good start. Happy to take more if you have ideas or thoughts on where/how it should fit in. Basically happy to brainstorm/discuss, but it may be better to ensure we are on the same page lest you get along a path that (for whatever reason) doesn't work out. Thanks!
